code/contact.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en"
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wo gibt's das Brot zu probieren?</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <a href="./">Willkommen!</a>
+</head>
+<body>
+    <h1>Kontakt</h1>
+    <p>Mein wöchentlicher Backtag ist freitags. Am Mittwoch vorher informiere ich meine Kunden über die geplante Produktion, damit sie ihr Wunschbrot bestellen können. So stelle sicher, dass ich nicht zu viel produziere.</p>
+    <p>Möchtest du einmal Brot probieren oder künftig bestellen? Sehr gerne nehme ich dich in meine Verteilerliste auf. Hinterlass mir doch einfach deine Kontaktdaten und du hörst bald von mir.</p>
+    <form>
+        <fieldset>
+            <legend>Kontakt</legend>
+            <div class="contact-details">
+                <label>Vorname</label>
+                <input class="contact-field" type="text" name="first-name">
+                <label>Nachname</label>
+                <input class="contact-field" type="text" name="last-name">
+                <label>Wohnort / evtl. Lieferort</label>
+                <input class="contact-field" type="text" name="city">
+                <label>Handynummer</label>
+                <input class="contact-field" type="tel" name="phone-no">              
+            </div>
+            <div class="permission">
+                <span>Darf ich dich per Whatsapp kontaktieren?</span>
+                <input class="contact-button" type="radio" name="WA-contact">
+                <label>Ja</label>
+                <input class="contact-button" type="radio" name="WA-contact">
+                <label>Nein</label>
+            </div>
+        </fieldset>
+    </form>

```

Comment:
I would suggest adding a submit button here as well, to be able to submit the form:
```
    <button type="Submit">Submit</button>
  </fieldset>
</form>
```